From 3ad6f1e6e95d5566f2278786f3c112fc4c991a10 Mon Sep 17 00:00:00 2001
From: Laith <laith@disroot.org>
Date: Mon, 9 Sep 2024 17:31:54 +0000
Subject: [PATCH] Modify colors in status_bar.c

---
 clients/status_bar.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/clients/status_bar.c b/clients/status_bar.c
index 3006e91..c103244 100644
--- a/clients/status_bar.c
+++ b/clients/status_bar.c
@@ -164,8 +164,8 @@ static const struct item_interface divider_interface = {
 /* Configuration parameters */
 static const int spacing = 12;
 static const char *const font_name = "Terminus:pixelsize=14";
-static const struct style normal = { .bg = 0xff1a1a1a, .fg = 0xff999999 };
-static const struct style selected = { .bg = 0xff338833, .fg = 0xffffffff };
+static const struct style normal = { .bg = 0xff282828, .fg = 0xffbdae93 };
+static const struct style selected = { .bg = 0xff3c3836, .fg = 0x458588ff };
 
 static timer_t timer;
 static bool running, need_draw;
-- 
2.44.0

